airflow/providers/amazon/aws/waiters/kinesisanalyticsv2.json

```diff
@@ -0,0 +1,151 @@
+{
+    "version": 2,
+    "waiters": {
+        "application_start_complete": {
+            "delay": 120,
+            "maxAttempts": 75,
+            "operation": "DescribeApplication",
+            "acceptors": [
+                {
+                    "matcher": "path",
+                    "argument": "ApplicationDetail.ApplicationStatus",
+                    "expected": "STARTING",
+                    "state": "retry"
+                },
+                {
+                    "matcher": "path",
+                    "argument": "ApplicationDetail.ApplicationStatus",
+                    "expected": "UPDATING",
+                    "state": "retry"
+                },
+                {
+                    "matcher": "path",
+                    "argument": "ApplicationDetail.ApplicationStatus",
+                    "expected": "AUTOSCALING",
+                    "state": "retry"
+                },
+                {
+                    "matcher": "path",
+                    "argument": "ApplicationDetail.ApplicationStatus",
+                    "expected": "DELETING",
+                    "state": "failure"
+                },
+                {
+                    "matcher": "path",
+                    "argument": "ApplicationDetail.ApplicationStatus",
+                    "expected": "STOPPING",
+                    "state": "failure"
+                },
+                {
+                    "matcher": "path",
+                    "argument": "ApplicationDetail.ApplicationStatus",
+                    "expected": "READY",
+                    "state": "failure"
```

Comment:
Out of curiosity, are you sure? `READY` does not feel a failure to me but you have more context than me

Reply:
Hi @vincbeck Yes, The `application_start_complete` waiter is used when the `KinesisAnalyticsV2StartApplicationOperator` is requested. If the start action does not reach the "running" state and fails to start, it returns to the "READY" state. Therefore, I am marking this as failed.  


I was initially planned to use the `describe_application_operation` https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/kinesisanalyticsv2/client/describe_application_operation.html  to get more details about the operation but boto3 version which is currently in provider not supporting as this api requires min boto3 1.34.134. https://github.com/boto/boto3/blob/develop/CHANGELOG.rst#L126